_source/api/logzio-public-api.yml

```diff
@@ -3575,10 +3581,22 @@ paths:
         - Lookup lists
       operationId: addLookupListElements
       summary: Add elements in bulk
-      description: Adds an array of elements to an existing Lookup list.
+      description: Adds an array of elements to an existing Lookup list and sets the expiration date for the lookup.
       produces:
         - application/json
       parameters:
+        - name: accountId
+          in: query
+          required: false
+          type: integer
+          format: int32
+          description:  Logz.io account ID
+        - name: defaultTTL
```

Comment:
@ohadza 

 What is the default value of TTL?
 How can a user configure the list so that it never expires?

Reply:
the default for defaultTTL is empty it also is never expires
for expirationTime null is never expires
BTW I consider adding to implementation that 0 also never expires, what do you think?

Reply:
after consulting with Yasmin I leave the implementation as it right now
Meaning no 0